app/views/questionnaire.py

```diff
@@ -184,7 +184,6 @@ def get_thank_you(eq_id, form_type, collection_id):  # pylint: disable=unused-ar
         return _redirect_to_latest_location(routing_path, collection_id, eq_id, form_type)
 
 
-@login_required
```

Comment:
Do we remove this as its now done in the path_finder_helper?

Reply:
Sort of. The step to redirect is a bit late to be enforcing the authentication. Its now been moved to the stop before where the path finder uses the `current_user` context